src/app/App.js

```diff
@@ -39,6 +40,12 @@ export const App = ({ options }) => {
     inside: mainContainer,
     withClasses: 'mainContainer__gameMode',
   });
+  const modeRules = render({
```

Comment:
Dobrze jakbyś to już podpięła, żeby działało :) Bo mamy zrobiony wybór trybu. Zrób np. chociaż zmianę tekstu zasad. A obrazkiem zajmiemy się kiedy indziej.

Reply:
Docelowy tekst jest zakomentowany, bo reszta komponentów się nie mieści, więc zostawiłam na razie wersję roboczą :)